keyboards/ashpil/modelm_usbc/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "IBM Model M 101 ANSI/102 ISO", 
+    "url": "https://github.com/iw0rm3r/qmk_firmware/tree/modelm101/keyboards/converter/modelm101", 
+    "maintainer": "ashpil", 
+    "width": 22.5, 
+    "height": 6.5, 
+    "layouts": {
+        "LAYOUT": {
```

Comment:
And this needs to be replaced with equivalents for the two layouts. I think you can just duplicate it, rename one of them to `LAYOUT_fullsize_iso_wkl` and remove the backslash, and rename the other to `LAYOUT_fullsize_ansi_wkl` and remove the `KC_NUBS` and `KC_NUHS` keys from that one.